site/src/docs/foundation/design-tokens/colour/index.mdx

```diff
@@ -33,6 +34,7 @@ The design system uses a colour theme that includes:
 - **Secondary and Tertiary colours:** these provide more ways for accenting select UI elements and distinguishing your service. The secondary and tertiary colours should be applied sparingly and with intent.
 - **UI colours:** the common colours for feedback (error, warning, success, info).
 - **Grayscale:** neutral colour shades.
+- **Focus colors:** the common colours for focus.
```

Comment:
We should either use _color(s)_ or _colour(s)_ in all of our texts, not both 🤓

Reply:
Seems that at least on the code-side it's color so colour(s) -> color(s) is the way to go

Reply:
Actually, we have agreed that the documentation in HDS follows British English, so colour is the correct one. In code color is usually special word so it cannot use British typing.

Reply:
ok... I'm more fond of American english :D :D